definitions/npm/@mapbox/mapbox-gl-supported_v2.x.x/flow_v0.25.x-/mapbox-gl-supported_v2.x.x.js

```diff
@@ -0,0 +1,8 @@
+declare module '@mapbox/mapbox-gl-supported' {
+  declare type OptionsT = {
+    failIfMajorPerformanceCaveat: boolean,
+    ...
+  };
+  declare export function supported(options?: OptionsT): boolean;
+  declare export function notSupportedReason(options?: OptionsT): ?string;
```

Comment:
Based on example from npm that recommends usage as

```
var mapboxglSupported = require('@mapbox/mapbox-gl-supported');

if (mapboxglSupported.supported()) {
    ...
}
```

Perhaps using `module.exports` will give more compatibility. This style should work without any changes to your tests which I assume is how you're consuming in your app.

```suggestion
  module.exports = {|
    supported: (options?: OptionsT) => boolean,
    notSupportedReason: (options?: OptionsT) => ?string,
  |};
```

Reply:
Yep, I think that makes sense. Updated.